web/src/views/auth/Login.vue

```diff
@@ -148,6 +148,10 @@ const login = async (form) => {
         password: loginUser.password,
         setCookie: true,
       }).then((response) => {
+        setUser({
+          name: loginUser.name,
+        })
+        localStorage.setItem('token',"bearer "+response.data.data.token)
```

Comment:
same with above. need not set token again.

Reply:
sorry, in my env,  If don't add set, when  login, it won't jump to the home page

Reply:
My guess is that something is wrong here.
```
router.beforeEach((to, from, next) => {
  let isAuthenticated = false;
  let user = getUser();
  if (user && user.name) {
    isAuthenticated = true;
  }

  if (!isAuthenticated && to.name !== 'Login' && to.name !== 'OAuth') next({ name: 'Login' })
  else if(isAuthenticated && (to.name == 'Login' || to.name == 'OAuth' )) next({ name: 'Index'})
  else next()
})

Reply:
I think `setUser` is necessary, you can test without `localStorage.setItem` and set token header

Reply:
This may a CORS issue, please guarantee the server host in `web/src/axios/index.js` is same with the `web/src/vite.config.js`

https://github.com/qingwave/weave/blob/529c56fc495a70fdf218b70f32c8c559b0e09368/web/vite.config.js#L47

Your code shows the server host in axios is `192.168.30.134` not the `127.0.0.1`

Reply:
OK,thanks for your answer, I try it again.